0-ad-history-mod-package/simulation/templates/units/han/champion_cavalry_spearman_academy.xml

```diff
@@ -8,6 +8,13 @@
     <Requirements>
       <Techs datatype="tokens">-unlock_champion_cavalry phase_city</Techs>
     </Requirements>
+    <History>
+        The Tiger and Leopard Cavalry was an elite cavalry division belonging to the warlord Cao Cao, commanded by his cousin Cao Chu. At that time, during the late Han dynasty and the early Three Kingdoms era (the first and second centuries BC), the use of cavalry was generally declining and shifting more towards heavier armor. There were other, similar cavalry divisions to the Tiger and Leopard cavalry in the armies of other warlords as well.
+        Admission into this special force had high requirements; applicants, for example, had to command at least a hundred units to be eligible. Because of that, both the commander, Cao Chun, and the squadron itself were renown for their elite status, extreme discipline, and physical capabilities. They could, for example, travel great distances in a comparably short amount of time.
+        The Tiger and Leopard Cavalry fought with the typical Chinese spears with leaf-shaped blades. They also carried a shield. And, as already indicated, horses and riders were heavily armored. Their equipment was of very high quality.
+        Despite this, one of the Tiger and Leopard Cavalry's strengths was their high flexibility in battle. They fought as shock troops and, at the desired moment and place on the battlefield, charged the enemy, trying to break up or disorganize their formation.
```

Comment:
This paragraph is confusing? Despite what? And yes a flexible force is cool, but you only give one example. (Shock troopers) What else could they do?

Reply:
I rearranged some sentences, I hope it makes more sense now. Despite their heavy armor. Flexible also because of their great travel speed. But yeah, its not a strength exclusive to this exact unit but to (especially professional) cavalry in general. If you think flexible doesnt fit here, you can remove it.